doc/playbook.rst

```diff
@@ -121,10 +121,20 @@ The bucket ``rubin-pp-dev-users`` holds:
 
 * ``rubin-pp-dev-users/unobserved/`` contains raw files that the upload script(s) can draw from to create incoming raws.
 
-``rubin-pp-dev`` has had notifications configured for it; these publish to a Kafka topic.
+``rubin-pp-dev`` has had notifications configured for it; these publish to the Kafka topic ``prompt-processing-dev``.
+The notifications can be viewed at `Kafdrop <https://k8s.slac.stanford.edu/usdf-prompt-processing-dev/kafdrop>`_.
+.
```

Comment:
Extra period. Should the Kafdrop link have been included in the previous commit?

Reply:
IIRC the "previous commit" was the commit changing all old bucket names to the new ones. As the Kafdrop setup is independent of the bucket change, to me it looks clearer to keep one commit purely about the bucket change, and another commit about updating the playbook.

Reply:
Ok, I thought it was the result of setting up the bucket.